db/migrate/20180517195006_remove_orphaned_conference_preferences.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveOrphanedConferencePreferences < ActiveRecord::Migration[5.1]
+  def up
+    execute <<~SQL
+      DELETE
+      FROM conference_preferences
+      WHERE first_conference_id IS NULL AND second_conference_id IS NULL
```

Comment:
Should we have a validation to prevent records like this from existing or is this not a problem at all?

I can't tell for sure if orphaned conference preferences is something with a "valid business meaning" - I have one in my local database that I created myself somehow (not seeded), so I assume their existence is expected (?)

Reply:
Oh yes, valid point @pgaspar 👍 
Rails 5 requires `belongs_to` associations by default. I thought I also turned it on for the Teams App, but apparently didn't 🙈 

I'll have a look _(the changes may be way out of the scope of this PR eventually though...)_